scipy/stats/_distn_infrastructure.py

```diff
@@ -2165,6 +2165,14 @@ def fit(self, data, *args, **kwds):
         >>> a1
         1
 
+        Not all distributions return estimates for the shape parameters.
+        ``norm`` for example just returns estimates for location and scale:
+
+        >>> from scipy.stats import norm
+        >>> x = norm.rvs(a, b, size=1000)
+        >>> loc1, scale1 = norm.fit(x)
+        >>> loc1, scale1
+        (0.98117775424344378, 2.09396423983832)
```

Comment:
You need to seed the generator. E.g. `rvs(,..., random_state=123)`. That'll fix the CI failure too.